rrmngmnt/host.py

```diff
@@ -251,12 +252,20 @@ def copy_to(self, resource, src, dst):
         :type src: str
         :param dst: path to destination
         :type dst: str
+        :param mode: file permissions
+        :type mode: str
+        :param ownership: file ownership(ex. ('root', 'root'))
+        :type ownership: tuple
         """
         with resource.executor().session() as resource_session:
             with self.executor().session() as host_session:
                 with resource_session.open_file(src, 'rb') as resource_file:
                     with host_session.open_file(dst, 'wb') as host_file:
                         host_file.write(resource_file.read())
+        if mode:
+            self.fs.chmod(path=dst, mode=mode)
+        if ownership:
+            self.fs.chown(path=dst, *ownership)
```

Comment:
I think you can not pass keyword and than *args  ... I think it will end with TypeError